theme/tuxlite_tbs/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```

Comment:
ugh, one of the worst licenses. That said, I don't think litigation or even derivative works are likely in the lifespan of this theme, so I'm fine with it for now.